src/test/ui/macro_backtrace/main.stderr

```diff
@@ -42,31 +42,31 @@ error: expected one of `!`, `.`, `::`, `;`, `?`, `{`, `}`, or an operator, found
 28 |       deep!();
    |       -------- in this macro invocation (#1)
    | 
-  ::: <deep macros>
+  ::: <deep macros>:1:1
```

Comment:
this doesn't really feel like a win *here*... but I guess it's ok? Is tan actual file line reflected here?

Reply:
We could check if `annotated_file.file.name` is `FileName::Real(path_buf)` to present the line and col, but honestly in reality I don't think it is going to be that much of a problem, and actually beneficial in real, verbose, macros.

Reply:
Yeah, I was ambivalent about this case: since it didn't seem to be decreasing readability, I decided not to special case it here. The line number is relative to the start of the macro, rather than the file in this case. It's also behind a `-Z` flag, so I thought it would affect people minimally regardless.